heaps/min_heap.py

```diff
@@ -17,25 +17,32 @@ class MinHeap:
     def __init__(self):
         self.store = []
 
-
     def add(self, key, value = None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
-        pass
```

Comment:
👍 However the space complexity is O(log n) due to `heap_up`'s recursive call stack.